SystemServices/SystemServicesHelper.cpp

```diff
@@ -391,6 +391,32 @@ string getTimeZoneDSTHelper(void)
     return timeZone;
 }
 
+/***
+ * @brief	: To retrieve TimeZone Accuracy
+ * @return	: string; TimeZone Accuracy
+ */
+string getTimeZoneAccuracyDSTHelper(void)
+{
+    string timeZoneAccuracy = "";
+    vector<string> lines;
+
+    if (!Utils::fileExists(TZ_ACCURACY_FILE)) {
+        return TZ_ACCURACY_INITIAL;
+    }
+
+    if (getFileContent(TZ_ACCURACY_FILE, lines)) {
+        if (lines.size() > 0) {
+            timeZoneAccuracy = lines.front();
+        }
+    }
+
+    if (timeZoneAccuracy != TZ_ACCURACY_INITIAL && timeZoneAccuracy != TZ_ACCURACY_INTERIM && timeZoneAccuracy != TZ_ACCURACY_FINAL) {
+        timeZoneAccuracy = TZ_ACCURACY_INITIAL;
```

Comment:
Pls add a log line with something like "Invalid timezone accuracy read from file, defaulting to "INITIAL" "